### PR TITLE
docs: Added Debian, RedHat, openSUSE, installation instructions

### PR DESCRIPTION
**Pull request by shenghuang147**
### Background

Some users don't know how to use systemd to start daed
图片

### Checklist

- [x] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/daed

### Full Changelogs

- [Implement ...]

### Issue Reference

Closes #_[issue number]_

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -26,3 +26,66 @@ sudo daed [-h,--help]
 If everything goes well, open your browser and navigate to `http://localhost:2023`
 
 Happy Hacking!
```
Comment (yqlbu) [inline docs/getting-started.md:28]: Please place this line at the end of the file.

**Review by yqlbu — CHANGES_REQUESTED** [review]

**Comment by shenghuang147** [comment]
I would also like to add an explanation of Arch if I can, but I don't know Arch well

``````shell
#yay -S daed
pacman -S --needed git base-devel
git clone https://aur.archlinux.org/daed.git
cd daed
makepkg -si
``````

This is my modification based on [dae](https://github.com/daeuniverse/dae/tree/main/docs/en#archlinuxmanjaro), can you see if this works?

File: docs/getting-started.md (hunk 1 above)
Comment (shenghuang147) [reply docs/getting-started.md]: see #246

**Comment by yqlbu** [comment]
> 

Any idea? @mzz2017 @kunish

**Review by yqlbu — APPROVED** [review]

**Comment by yqlbu** [comment]
> I would also like to add an explanation of Arch if I can, but I don't know Arch well
> 
> ```shell
> #yay -S daed
> pacman -S --needed git base-devel
> git clone https://aur.archlinux.org/daed.git
> cd daed
> makepkg -si
> ```
> 
> This is my modification based on [dae](https://github.com/daeuniverse/dae/tree/main/docs/en#archlinuxmanjaro), can you see if this works?

Any idea? @mzz2017 @kunish

**Comment by kunish** [comment]
Looking good in general, I understand this is a WIP

**Comment by yqlbu** [comment]
@kunish Let's merge it then.

File: docs/getting-started.md [hunk 2]
```diff
@@ -25,4 +25,67 @@ sudo daed [-h,--help]
...
+rm /tmp/installer-daed-linux-$(arch).deb
+
+# Start daed
+sudo systemctl start daed
+
+# enable daed start automatically
+sudo systemctl enable daed
+``````
+
+If everything goes well, open your browser and navigate to `http://localhost:2023`
```
Comment (kunish) [inline docs/getting-started.md:47]: Maybe we dont need to mention this in every installation

Comment (kunish) [reply docs/getting-started.md:47]: Feels redundant

File: docs/getting-started.md [hunk 3]
```diff
@@ -25,4 +25,67 @@ sudo daed [-h,--help]
...
+rm /tmp/installer-daed-linux-$(arch).rpm
+
+# Start daed
+sudo systemctl start daed
+
+# enable daed start automatically
+sudo systemctl enable daed
+``````
+
+If everything goes well, open your browser and navigate to `http://localhost:2023`
```
Comment (yqlbu) [inline docs/getting-started.md:68]:
```suggestion
```

**Review by yqlbu — CHANGES_REQUESTED** [review]

**Comment by kunish** [comment]
@yqlbu No need to rush, lets do some minor cleanings first

File: docs/getting-started.md [hunk 4]
```diff
@@ -25,4 +25,65 @@ sudo daed [-h,--help]
...
```
Comment (kunish) [inline docs/getting-started.md:87]:
```suggestion
```

**Review by kunish — APPROVED** [review]
LGTM

**Review by yqlbu — APPROVED** [review]
LGTM.

**Comment by mzz2017** [comment]
> https://aur.archlinux.org/daed.git
> cd daed

LGTM. Please also add AUR. Thanks.

**Pull request merged**